check_replication_queue.py

```diff
@@ -0,0 +1,32 @@
+from client import get_client
+import sys
+import waiting
+import click
+
+
+SNUBA_ADMIN_HOST = "127.0.0.1"
+CLICKHOUSE_CLIENT_PORT = "9000"
+
+def _is_replication_queue_empty(client):
+    return len(client.execute("SELECT * FROM system.replication_queue LIMIT 1;")) == 0
+
+
+@click.command()
+@click.option("--host", default=SNUBA_ADMIN_HOST)
+def check_replication_queue(host: str):
+    client = get_client(host, CLICKHOUSE_CLIENT_PORT)
+    try:
+        waiting.wait(lambda: _is_replication_queue_empty(client), sleep_seconds=60, timeout_seconds=300)
```

Comment:
I think we could make the timeout seconds and sleep seconds be options as well and you can use the values you have as defaults